system_strings/sysmes_text.en

```diff
@@ -5459,159 +5515,255 @@ Death calls out to me. There's^nowhere left to run. At that^moment――
+// @flowchart_descr
+The day dawns on a sleepless night.
+ The wound in my right hand has already healed.
+ This body certainly has its perks.
 }
 [sha:4c5b1a100d8219c54eb0ca82036e47dac1e18654]{
 -- 時間の経過がおかしい。^萎びていく意識と裏腹に、^身体は衝動を持て余していた。
-時間の経過がおかしい。^萎びていく意識と裏腹に、^身体は衝動を持て余していた。
+// @flowchart_descr
+Time is passing in fits and starts.
+ Even as my consciousness begins to wither and fade,
+ my body is overwhelmed with urges.
 }
 [sha:54711c3cf9e42db20dcaed5431b08270de3dc298]{
 -- 扉が開いていく。^琥珀が部屋に入ってくる。^最後の理性で警告をするが―――
-扉が開いていく。^琥珀が部屋に入ってくる。^最後の理性で警告をするが―――
+// @flowchart_descr
+The door opens.
+ Kohaku begins to enter my room.
+ With the last of my sanity, I warn her, but―――
 }
 [sha:cab52b01c87504ee57165fe24d891193af5a5b74]{
 -- 体中がたぎっている。^食事を置いたなら、すぐにでも、^出て行ってもらわなければ……。
-体中がたぎっている。^食事を置いたなら、すぐにでも、^出て行ってもらわなければ……。
+// @flowchart_descr
+My body is like a wound spring.
+ I need her to get out of here as soon as she leaves the food―――
 }
 [sha:ae4ab28f7b12ad0942b6dc0a7ff34bdf2ac2d825]{
 -- もう歯止めは利かなかった。^喉が渇いてたまらない。^ふと我に返ると、そこには―――
-もう歯止めは利かなかった。^喉が渇いてたまらない。^ふと我に返ると、そこには―――
+// @flowchart_descr
+I just couldn't hold myself back any longer.
+ I was so thirsty I couldn't bear it.
+ When I come to my senses, I'm greeted by―――
 }
 [sha:87efc66e65a3aa31f879cf5d0dca46afa3bb12d4]{
 -- 屋敷の外に出る。^また堪えきれなくなる前に、^誰もいない場所へ逃げなければ。
-屋敷の外に出る。^また堪えきれなくなる前に、^誰もいない場所へ逃げなければ。
+// @flowchart_descr
+I flee the mansion.
+ I need to escape to somewhere without anyone else around before I lose control again.
 }
 [sha:ecdbf9cab2a69d966ccfc4a564a8546bd0a7fccb]{
 -- デパートの跡地に潜り込む。^きっと先輩は来てくれる。^来てくれる。
-デパートの跡地に潜り込む。^きっと先輩は来てくれる。^来てくれる。
+// @flowchart_descr
+I sneak into the ruins where the department store once stood.
+ Senpai will definitely come.
+ I'm certain of it.
 }
 [sha:46b58330c3f04e233512b1d2767a4435b8dfb655]{
 -- 背中を見せれば撃たれるだけだ。^全力で前に向かって走り出す。^そのまま真横をすり抜けて―――
-背中を見せれば撃たれるだけだ。^全力で前に向かって走り出す。^そのまま真横をすり抜けて―――
+// Quoting 33900, 33907.
+// @flowchart_descr
+If I tried to run away, she'd run my heart through the moment I turned my back on her.
+ Instead I lunge forward as fast as I can, and slip through her flank―――
 }
 [sha:698e2c94f7986733faead7a248e098f2db225785]{
 -- 背中を向けて走る。^今の肉体なら、^難なく逃げだせる筈だ―――
-背中を向けて走る。^今の肉体なら、^難なく逃げだせる筈だ―――
+// @flowchart_descr
+I'll turn tail and run.
+ With my transformed body, I should have no problem escaping from her―――
 }
 [sha:c82da45a8c696a0a8996bcbd14535fd25d79e011]{
 -- これ以上逃げられそうにない。^すぐに彼女がやってくるだろう。^まだ生きたいと思うのなら――
-これ以上逃げられそうにない。^すぐに彼女がやってくるだろう。^まだ生きたいと思うのなら――
+// @flowchart_descr
+It looks like this is as far as I can run.
+ I'm sure she'll be here any second now.
+ If I still want to live, I have to―――
 }
 [sha:c631f090e0b297f8b9f4e2c4ecf257e0df8cae21]{
 -- すべて幻になってしまえばいい。^ナイフを握って立ち上がり、^最後の代行者と対峙する。
-すべて幻になってしまえばいい。^ナイフを握って立ち上がり、^最後の代行者と対峙する。
+// Quoting 34277.
+// @flowchart_descr
+If it all becomes nothing more than an illusion, that's fine.
+ Gripping my knife firmly, I stand to face the Executor one last time.
```

Comment:
34277 is kinda non-ideal, imo.
>I don't mind it all fading into illusion.

Reply:
Don't really love that either. Feels kinda mild. Maybe one of
>  So what if it all becomes a mere illusion?             
> I don't care if it all becomes nothing but an illusion.

Reply:
I kinda like 'fade' because then it works with the opposing choice

>So what if it all fades into illusion?